src/main/java/seedu/address/commons/util/StringUtil.java

```diff
@@ -65,4 +65,22 @@ public static boolean isNonZeroUnsignedInteger(String s) {
             return false;
         }
     }
+
+    /**
+     * Returns true if {@code s} the index is valid number
+     * e.g. delt 1
+     * Will return false for any other invalid inout
+     * e.g. empty string, delt abc
+     * @throws NullPointerException if {@code s} is null.
+     */
+    public static boolean isInvalidNumber(String s) {
+        requireNonNull(s);
+
+        try {
+            Integer a = Integer.parseInt(s.replaceAll("[^0-9]", ""));
```

Comment:
Would you consider a mix of alphabets and number as invalid number? E.g. `a2` will become `2` after this replace. However is it intended for the command to work when the user typed `delt a2`?

Reply:
Fixed with update